src/Microsoft.PowerShell.Commands.Management/commands/management/GetChildrenCommand.cs

```diff
@@ -172,6 +172,23 @@ public uint Depth
         }
 
         /// <summary>
+        /// Gets or sets whether symbolic links are to be followed when recursing.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter FollowSymlink
+        {
+            get
+            {
+                return _followSymlink;
+            }
+            set
+            {
+                _followSymlink = value;
```

Comment:
Modified the `-FollowSymlink` parameter to set `-Recurse`, and the `-Depth` parameter does. Thus we cannot use auto-property here.